utils/generate_dbt/generate_dbt.py

```diff
@@ -96,7 +104,8 @@ def generate_sql_files(
         columns_for_hashid = [
             f"'{field[0]}'"
             for field in columns
-            if field[1] in ["STRING", "INT64", "FLOAT64", "BOOL", "DATE", "TIMESTAMP"] and not field[0].startswith("_airbyte")
+            if field[1] in ["STRING", "INT64", "FLOAT64", "BOOL", "DATE", "TIMESTAMP"]
```

Comment:
We should probably think about how to make this work for the nested and repeated field types. Definitely not a rn thing, but i think a possible edge case is that we could have a table where all of the fields we are creating the hash id with could be the same for multiple records but then they have unique columns that were just not including in the hash field (json cols). So it would count it as a dupe when not really a dupe. Does that make sense? or am i overthinking this?